src/store/reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { Survey } from 'services/survey';
+import { StoreType } from 'store';
+
+export type ActionType = {
+  type: 'setSurveys';
+  value: Survey[];
+};
+
+export default (store: StoreType, action: ActionType) => {
+  switch (action.type) {
+    case 'setSurveys':
```

Comment:
How about moving the action name to a separate file, maybe putting it under the `constants` folder? so that we can reuse it everywhere to avoid misspelling and make it easier to maintain 💭

Reply:
Updated to be enum [629cd85](https://github.com/hanam1ni/react-surveyor/pull/68/commits/629cd850c4e768eb2b4f94d7eb5f60986a91bbe1)